example.py

```diff
@@ -59,13 +60,21 @@ def main():
     setup = Setup(run_pipeline)
     # Specify the constant parameters for evaluation
     setup.parameter('data', data)
-    setup.parameter('output_dir', 'scratch')
+    # Specify the main output_dir
+    main_output_dir = Path('scratch')
 
     # Specify the variable parameters for evaluation
-    for _, parameters, predicted_lrs in setup.run_full_grid({'model': models}):
+    for variabele, parameters, (predicted_lrs, y_true) in \
+            setup.run_full_grid({'model': models}):
         model_name = parameters['model'].__class__.__name__
         print(f"{model_name}: {predicted_lrs}")
 
+        output_dir = main_output_dir / ''.join(list(variabele.keys()))
```

Comment:
Using just the keys, multiple combinations of variables will still result in the same string right? Would something like the following work?

```suggestion
        output_dir = main_output_dir / '_'.join(f'{key}-{value}' for key, value in variabele.items())
```

(something like `.../model-a_param-b`)

Reply:
wordt nu zo gedaan